azure/scope/cluster_test.go

```diff
@@ -2005,7 +2005,7 @@ func TestOutboundLBName(t *testing.T) {
 			clusterName: "my-cluster",
 			name:        "public cluster node outbound lb",
 			role:        "node",
-			expected:    "my-cluster",
+			expected:    "",
```

Comment:
Just for my own understanding: Why is this change needed?

Reply:
BTW, this also is the reason why the `loadBalancerName` is changed from `foo` to `""` for test cases in controllers/helpers_test.go file.